packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -52,7 +52,7 @@ const WorkspaceHeader: React.FC<{}> = () => {
       (isResourceRoute && currentResource) ||
       (isResourceRoute && currentProjectConfiguration)
     )
-      return currentResource?.name || currentProjectConfiguration?.name;
+      return currentResource?.resourceType ===  models.EnumResourceType.Service ?  currentResource?.name : 'Project Configuration'
```

Comment:
also, after updating the resource name from the client, we need to refetch the project list because with now the project name is not refreshed on the UI

Reply:
Done